static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,218 @@
+# Data Storage on External Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data are huge, which means that they won't fit on our home
+directory, and even if they did, it would certainly take a long time to copy
```

Comment:
`they -> it` and a few other place to fix accordingly

Reply:
Already fixed.